### ISSUE TITLE
Can not use convert2df() with Scopus file

### ISSUE THREAD

**Issue by akastrin** [massimoaria/bibliometrix#19]
I try to convert Scopus RIS file to data frame using the command:
```
D <- readFiles("~/Desktop/scopus.ris")
M <- convert2df(D, dbsource = "scopus", format = "plaintext")
```
Unfortunately, I get the error:
`Error in seq.default(iP, iPs) : 'to' must be a finite number`

I try with default version of `bibliometrix` on CRAN and with recent GitHub version. Any idea what's wrong?

Andrej

**Comment by akastrin** [comment]
Thanks. That works.

**Comment by massimoaria** [comment]
A file ".ris" is not in the right format.


You have to download your collection from Scopus using bibtex export format file.




Ottieni Outlook per Android







On Sat, Oct 27, 2018 at 12:36 PM +0200, "akastrin" <notifications@github.com> wrote:












Closed #19.
